oxigen_stock_barcodes_gs1/wizard/stock_barcodes_read.py

```diff
@@ -155,6 +155,9 @@ def process_barcode(self, barcode):  # noqa: C901
         if processed:
             if not self.check_option_required():
                 return False
+            if self.is_manual_confirm or self.manual_entry:
+                self._set_messagge_info("info", _("Review and confirm"))
+                return False
```

Comment:
@FrankC013 Is this a bug intriduced by us or it's from the original OCA module???

Reply:
@eantones It is a bug introduced by us and also not handled by the original OCA module